### DISCUSSION TITLE
左侧资源管理器自定义组件增加右键菜单注册

### DISCUSSION THREAD

**Discussion by coulis1018** [opensumi/core#2013] [Q&A]
如图，根据案例在资源管理器中自定义todolist 组件，想要实现右键菜单注册功能：
<img width="703" alt="image" src="https://user-images.githubusercontent.com/54612727/203755705-2b46b3b1-666b-4e27-8489-0218b9fb77dd.png">
<img width="784" alt="image" src="https://user-images.githubusercontent.com/54612727/203756641-2480cb3a-7ef1-4abb-b781-e0b815bdda88.png">
应该对应哪个meunId呢？

**Comment by erha19** [comment]
@coulis1018 写了个简单例子，可以参考一下 https://github.com/opensumi/opensumi-module-samples/pull/15.
